### PR TITLE
DOCSUP-9861: Date and time operator update

### PR DESCRIPTION
**Pull request by kirillikoff**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #3931

### CODE REVIEW

File: docs/en/sql-reference/operators/index.md [hunk 1]
```diff
@@ -188,6 +188,24 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
 └─────────────────────┴────────────────────────────────────────────────────────────┘
 ```
 
+You can set the period without using `INTERVAL` by multiplying seconds, minutes, and hours. For example, a period of one day can be set at `60*60*24`.
+
+!!! note "Note"
+    Syntax `now() + 60*60*24` doesn't consider time settings. For example, daylight saving time. The `INTERVAL` syntax or `addDays`-function is always preferred.
```
Comment (olgarev) [inline docs/en/sql-reference/operators/index.md:194]:
```suggestion
    Syntax `now() + 60*60*24` doesn't consider time settings. For example, daylight saving time. The `INTERVAL` syntax or `addDays` function are always preferred.
```

File: docs/ru/sql-reference/operators/index.md [hunk 2]
```diff
@@ -189,6 +189,23 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
 └─────────────────────┴────────────────────────────────────────────────────────────┘
 ```
 
+Вы можете задать период без использования синтаксиса `INTERVAL`, умножив секунды, минуты и часы. Например, период в один день можно записать как `60*60*24`.
+
+!!! note "Примечание"
+    Синтаксис `now() + 60*60*24` не учитывает региональные настройки времени. Например, переход на летнее время. Для работы с интервалами синтаксис `INTERVAL` или функция `addDays` предпочтительней.
```
Comment (olgarev) [inline docs/ru/sql-reference/operators/index.md:195]:
```suggestion
    Для работы с интервалами лучше использовать синтаксис `INTERVAL` или функцию `addDays`. Простое сложение (например, `now() + ...`) не учитывает региональные настройки времени, например, переход на летнее время. 
```

File: docs/en/sql-reference/operators/index.md [hunk 3]
```diff
@@ -188,6 +188,24 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
 └─────────────────────┴────────────────────────────────────────────────────────────┘
 ```
 
+You can set the period without using `INTERVAL` by multiplying seconds, minutes, and hours. For example, a period of one day can be set at `60*60*24`.
```
Comment (olgarev) [inline docs/en/sql-reference/operators/index.md:191]:
```suggestion
You can set the date without using `INTERVAL`, just by adding or subtracting seconds, minutes, and hours. For example, an interval of one day can be set by adding `60*60*24`.
```

File: docs/ru/sql-reference/operators/index.md [hunk 4]
```diff
@@ -189,6 +189,23 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
 └─────────────────────┴────────────────────────────────────────────────────────────┘
 ```
 
+Вы можете задать период без использования синтаксиса `INTERVAL`, умножив секунды, минуты и часы. Например, период в один день можно записать как `60*60*24`.
```
Comment (olgarev) [inline docs/ru/sql-reference/operators/index.md:192]:
```suggestion
Вы можете изменить дату, не используя синтаксис `INTERVAL`, а просто добавив или отняв секунды, минуты и часы. Например, чтобы передвинуть дату на один день вперед, можно прибавить к ней значение `60*60*24`.
```

File: docs/ru/sql-reference/operators/index.md [hunk 5]
```diff
@@ -189,6 +189,23 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
 └─────────────────────┴────────────────────────────────────────────────────────────┘
 ```
 
+Вы можете изменить дату, не используя синтаксис `INTERVAL`, а просто добавив или отняв секунды, минуты и часы. Например, чтобы передвинуть дату на один день вперед, можно прибавить к ней значение `60*60*24`.
+
+!!! note "Примечание"
+    Для работы с датами лучше использовать синтаксис `INTERVAL` или функцию `addDays`. Простое сложение (например, синтаксис `now() + ...`) не учитывает региональные настройки времени, например, переход на летнее время.
```
Comment (vdimir) [inline docs/ru/sql-reference/operators/index.md:195]:
```suggestion
    Синтаксис `INTERVAL` или функция `addDays` предпочтительнее для работы с датами. Сложение с числом (например, синтаксис `now() + ...`) не учитывает региональные настройки времени, например, переход на летнее время. 
```

**Review by vdimir — APPROVED** [review]

File: docs/en/sql-reference/operators/index.md [hunk 6]
```diff
@@ -188,6 +188,24 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
...
+You can work with dates without using `INTERVAL`, just by adding or subtracting seconds, minutes, and hours. For example, an interval of one day can be set by adding `60*60*24`.
+
+!!! note "Note"
+    The `INTERVAL` syntax or `addDays` function are always preferred. Simple addition or subtraction (syntax like `now() + ...`) doesn't consider time settings. For example, daylight saving time.
+
+
+Examples:
+
+``` sql
+SELECT now() AS current_date_time, current_date_time + 60*60*96;
```
Comment (KochetovNicolai) [inline docs/en/sql-reference/operators/index.md:200]: What if we add better example with time shift?
```sql
:) SELECT toDateTime('2014-10-26 00:00:00', 'Europe/Moscow')  AS time, time + 60*60*24 as time_plus_24_hours, (time + interval 1 day) as time_plus_1_day

SELECT
    toDateTime('2014-10-26 00:00:00', 'Europe/Moscow') AS time,
    time + ((60 * 60) * 24) AS time_plus_24_hours,
    time + toIntervalDay(1) AS time_plus_1_day

Query id: aaf32215-aa05-4a37-a370-b85b73aaf386

┌────────────────time─┬──time_plus_24_hours─┬─────time_plus_1_day─┐
│ 2014-10-26 00:00:00 │ 2014-10-26 23:00:00 │ 2014-10-27 00:00:00 │
└─────────────────────┴─────────────────────┴─────────────────────┘
```

(`2014-10-26 02:00:00` msk was time shift -1 hour)

**Review by KochetovNicolai — APPROVED** [review]

File: docs/en/sql-reference/operators/index.md (hunk 6 above)
Comment (kirillikoff) [reply docs/en/sql-reference/operators/index.md:200]: Great example, thanks.

Comment (vdimir) [inline docs/en/sql-reference/operators/index.md:200]:
```suggestion
SELECT
    toDateTime('2014-10-26 00:00:00', 'Europe/Moscow') AS time,
    time + 60 * 60 * 24 AS time_plus_24_hours,
    time + toIntervalDay(1) AS time_plus_1_day
```

File: docs/en/sql-reference/operators/index.md [hunk 7]
```diff
@@ -188,6 +188,24 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
...
+```
+
+``` text
+┌───current_date_time─┬─plus(now(), multiply(multiply(60, 60), 96))─┐
+│ 2021-06-12 17:34:49 │                         2021-06-16 17:34:49 │
+└─────────────────────┴─────────────────────────────────────────────┘
```
Comment (vdimir) [inline docs/en/sql-reference/operators/index.md:206]:
```suggestion
┌────────────────time─┬──time_plus_24_hours─┬─────time_plus_1_day─┐
│ 2014-10-26 00:00:00 │ 2014-10-26 23:00:00 │ 2014-10-27 00:00:00 │
└─────────────────────┴─────────────────────┴─────────────────────┘
```

File: docs/ru/sql-reference/operators/index.md [hunk 8]
```diff
@@ -189,6 +189,23 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
...
+Пример:
+
+``` sql
+SELECT now() AS current_date_time, current_date_time + 60*60*96;
+```
+
+``` text
+┌───current_date_time─┬─plus(now(), multiply(multiply(60, 60), 96))─┐
+│ 2021-06-12 17:34:49 │                         2021-06-16 17:34:49 │
+└─────────────────────┴─────────────────────────────────────────────┘
```
Comment (vdimir) [inline docs/ru/sql-reference/operators/index.md:206]:
```suggestion
┌────────────────time─┬──time_plus_24_hours─┬─────time_plus_1_day─┐
│ 2014-10-26 00:00:00 │ 2014-10-26 23:00:00 │ 2014-10-27 00:00:00 │
└─────────────────────┴─────────────────────┴─────────────────────┘
```

File: docs/ru/sql-reference/operators/index.md [hunk 9]
```diff
@@ -189,6 +189,23 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
...
 
+Вы можете изменить дату, не используя синтаксис `INTERVAL`, а просто добавив или отняв секунды, минуты и часы. Например, чтобы передвинуть дату на один день вперед, можно прибавить к ней значение `60*60*24`.
+
+!!! note "Примечание"
+    Синтаксис `INTERVAL` или функция `addDays` предпочтительнее для работы с датами. Сложение с числом (например, синтаксис `now() + ...`) не учитывает региональные настройки времени, например, переход на летнее время. 
+
+Пример:
+
+``` sql
+SELECT now() AS current_date_time, current_date_time + 60*60*96;
```
Comment (vdimir) [inline docs/ru/sql-reference/operators/index.md:200]:
```suggestion
SELECT
    toDateTime('2014-10-26 00:00:00', 'Europe/Moscow') AS time,
    time + 60 * 60 * 24 AS time_plus_24_hours,
    time + toIntervalDay(1) AS time_plus_1_day
```

**Pull request merged**